z3-sys/build.rs

```diff
@@ -17,6 +19,33 @@ fn main() {
     };
     println!("cargo:rerun-if-env-changed={}", Z3_HEADER_VAR);
     println!("cargo:rerun-if-changed={}", header);
+
+    generate_binding(&header);
+}
+
+#[cfg(feature = "vcpkg")]
+fn main() {
+    let lib = vcpkg::Config::new()
+        .emit_includes(true)
+        .find_package("z3")
+        .unwrap();
+    let found_header = lib.include_paths.iter().any(|include| {
```

Comment:
Does this actually have more than one include path present?

Reply:
Normally there will be exactly one. I just put it here in case something changes in the future.